pkg/relayer/session/interface.go

```diff
@@ -0,0 +1,53 @@
+package session
+
+import (
+	"github.com/pokt-network/smt"
+
+	"github.com/pokt-network/poktroll/pkg/observable"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+)
+
+// RelayerSessions is an interface for managing the relayer's sessions and Sparse Merkle Sum Trees (SMSTs).
+// It provides notifications about closing sessions that are ready to be claimed, and handles the creation
+// and retrieval of SMSTs for a given session. It can also be thought of as a SessionManager but dedicated
+// to the Relayer behavior.
```

Comment:
Is `SessionManager` still a thing? The last sentence raises more questions than it does answers for me. Could we revise it to avoid referencing unrelated or nonexistent objects, or just drop it altogther?

Wdyt about also including expiring sessions in the comment if we're going to mention closing sessions, e.g.:

```suggestion
// RelayerSessions is an interface for managing the relayer's sessions and Sparse
// Merkle Sum Trees (SMSTs). It provides notifications about closing and expiring
// sessions that are ready to be claimed and proven, respectively. It also handles
// the creation and retrieval of SMSTs for a given session.
```

Reply:
Expiring sessions is removed on #105